config/production.yaml

```diff
@@ -20,20 +20,8 @@ gcp:
     table:
     fields:
 
```

Comment:
remove that empty line?

Reply:
totally. I had local changes and missed this 👍